services/user-tenant-service/src/main/java/com/sourcefuse/jarc/services/usertenantservice/enums/PermissionKey.java

```diff
@@ -0,0 +1,57 @@
+package com.sourcefuse.jarc.services.usertenantservice.enums;
+
+public enum PermissionKey {
```

Comment:
this should be moved to core, secondly why the numbers in string why cant we remove them

Reply:
done and moved to core